src/update_agent/mod.rs

```diff
@@ -37,6 +38,10 @@ const DEFAULT_POSTPONEMENT_TIME_SECS: u64 = 60; // 1 minute.
 /// before abandoning a target update.
 const MAX_DEPLOY_ATTEMPTS: u8 = 12;
 
+/// This is undocumented; it's for testing Zincati in e.g. cosa where you have
+/// an interactive session but you don't want to wait for the full timeout.
+const INTERACTIVE_SESSION_OVERRIDE: &str = "/run/zincati/override-interactive-check";
```

Comment:
Seems OK as is, but I personally would have probably leaned towards making this an environment variable and told people to `systemctl edit --runtime zincati` instead.

Reply:
Yeah, I debated on this a bit. It's just... so much easier to touch a file than to type out a dropin. Though we could also have added a `cosa run --add-ignition zincati-override-interactive-check` to help.